### ISSUE TITLE
Marketplace: add PluginsBrowser e2e tests

### ISSUE THREAD

**Issue by allilevine** [Automattic/wp-calypso#66924]
We're missing tests for a lot of Marketplace (/plugins) functionality.

**Comment by lsl** [comment]
@lupus2k was https://github.com/Automattic/wp-calypso/pull/67125 20% time or something? Not seeing acceptance criteria or a sprint assigned to the issue. Wondering what to do with your PR as I triage the old sprint / board.

**Comment by lupus2k** [comment]
Yes, it was a [20% time](https://explorersp2.wordpress.com/2022/08/30/20-time-e2e-multisite-tests/)

**Comment by lupus2k** [comment]
Yes, it was a 20% time: pb5gDS-20e-p2

**Comment by lsl** [comment]
Ah, it even has the label :smile: thanks for the post.
